docs/backend-requests/versioning/available-versions.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Available Versions
+description: A list of the available API versions and their breaking changes.
+---
+
+# Available Versions
+
+Below is a list of all the available API Versions with their respective breaking changes. 
+
+### 2021-02-05
+
+The initial API version was released on 2021-02-05 as part of the first Clerk release. It predates the versioning system, all changes made to it since then are backwards compatible.
```

Comment:
```suggestion
The initial API version was released on 2021-02-05 as part of the first Clerk release. It predates the versioning system. All changes made to it since then are backwards compatible.
```